### DISCUSSION TITLE
General: it's a pity that a lot of functionalities and wizards for Jakarta EE, especially JSF (Faces), that worked before, do not work anymore for the last versions of Jakarta EE in NetBeans

### DISCUSSION THREAD

**Discussion by richard-grin** [apache/netbeans#5510] [General]
### Apache NetBeans version

Apache NetBeans 17 release candidate

### What happened

Some of the issues: #5370, #4892, #4839, #4830, #4828, #4827, #4829, #3659.
They appeared with the last versions of NetBeans (from the 15th version if I remember well) and Jakarta EE, for instance with Jakarta EE 10.
Each issue was planned to be fixed in the "next" version of NetBeans but, each time, the fix is postponed.

NetBeans was the best IDE for Java EE and the first versions of Jakarta EE, especially for JSF, but it is no longer the case.

I know there is no obvious solution for this problem but I wanted to raise the alarm. I can't help for the code of NetBeans but I can help to explain each problem in details if it is necessary.

### Did this work correctly in an earlier version?

No / Don't know

### Operating System

Windows 11 Family 64-bit, version 21H2

### JDK

17 11

### Apache NetBeans packaging

Apache NetBeans provided installer

### Are you willing to submit a pull request?

No

### Code of Conduct

Yes

**Comment by neilcsmith-net** [comment]
> Each issue was planned to be fixed in the "next" version of NetBeans but, each time, the fix is postponed.
>
> NetBeans was the best IDE for Java EE and the first versions of Jakarta EE, especially for JSF, but it is no longer the case.
>
> I know there is no obvious solution for this problem but I wanted to raise the alarm.

Converted this to a discussion, although the points are valid.

It's not entirely correct to say that the issues were planned to be fixed in the next version of NetBeans, or postponed.

The solution is obvious, just not necessarily easy, and that is that we need more developers willing to join in and put time and effort into fixing this area of the IDE.
